components/database/mongodb-40/patches/04-SConstruct.patch

```diff
@@ -0,0 +1,49 @@
+--- mongodb-src-r4.0.1/SConstruct.~1~	2018-07-27 20:14:20.000000000 +0000
++++ mongodb-src-r4.0.1/SConstruct	2018-08-12 20:35:00.869732979 +0000
+@@ -16,6 +16,8 @@
+ 
+ import SCons
+ 
++import psutil
++
+ # This must be first, even before EnsureSConsVersion, if
+ # we are to avoid bulk loading all tools in the DefaultEnvironment.
+ DefaultEnvironment(tools=[])
+@@ -35,6 +37,8 @@
+ 
+ import libdeps
+ 
++SetOption('num_jobs', max(1,psutil.cpu_count()-1))
++
```

Comment:
Can we set it in Makefile?

Reply:
Yes, if I would know a method how put the output of
CORES=´psrinfo -v|grep '^Status of'|tail -1|awk '{print $5}'`           does not work!
into a variable in the Makefile
COMPONENT_BUILD_ARGS += -j $(CORES)